src/state/apis/swapper/helpers/getInputOutputRatioFromQuote.ts

```diff
@@ -73,8 +73,9 @@ const _convertCryptoBaseUnitToUsdPrecision = (
   amountCryptoBaseUnit: BigNumber.Value,
 ): BigNumber => {
   const usdRate = selectUsdRateByAssetId(state, asset.assetId)
-  if (usdRate === undefined) throw Error(`missing usd rate for assetId ${asset.assetId}`)
-  return bnOrZero(fromBaseUnit(amountCryptoBaseUnit, asset.precision)).times(usdRate)
+  // TODO(gomes): revert me once we upsert market data JIT
```

Comment:
For now, this allows to get quotes for all routes, even those without market-data 🎉

Reply:
JIT not applicable anymore, update comment